src/Featurizers/Components/StatisticsMetricsEstimator.h

```diff
@@ -166,8 +166,9 @@ class StandardStatsUpdater : public BasicStatsUpdater<T> {
     // ----------------------------------------------------------------------
     typename TypeSelector::SumTypeSelector<T>::type                               _sum;
 };
+}

```

Comment:
The name of the type in on 111 is different from the name on 116.

Reply:
Good catch! It's fixed now.